### PR TITLE
stand up a jaseci_ai_kit server locally

### PR DESCRIPTION
**Pull request by Shawn-Jemmott**
@ypkang @marsninja

### CODE REVIEW

File: docs/docs/canonicai/chapter9.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Stand up a jaseci_ai_kit server locally
+
+In this tutorial, we will discuss how to use uvicorn to stand up a jaseci_ai_kit server locally and quickly
+
+Are you Excited? Let's jump right in.
+
+# Process
+First, you will have to cd into the path of the jaseci_ai_kit module for example let try to stand up cl_summer AI module. From the root jaseci directory run this command in the terminal as follows:
```
Comment (ypkang) [inline docs/docs/canonicai/chapter9.md]: Hey @Shawn-Jemmott l suggest you do a round of proof read after initial draft for this and any future writing contributions. Let's make sure there aren't any grammar issues and sentences and punctuations are used properly etc.

File: docs/docs/canonicai/chapter9.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* `cd jaseci_kit/jaseci_kit/modules/cl_summer`
+
+You can CD into any AI models, after you go into the cl_summer directory you will have to stand it up locally and how you do that you will run the following:
+
+* `uvicorn [name_of_file]:serv_actions ` in this case `uvicorn cl_summer:serv_actions`
```
Comment (ypkang) [inline docs/docs/canonicai/chapter9.md:44]: So this is one way to run the uvicorn. The other way, which I think we should recommend more, is to run it through the installed jaseci_ai_kit pip package. 

So if you have the jaseci_ai_kit python package installed, you can actually run it like this anywhere without having to be in the jaseci repo
```
uvicorn jaseci_ai_kit.tfm_ner:serv_actions
```

Comment (ypkang) [reply docs/docs/canonicai/chapter9.md:44]: Let's include this in here too.

File: docs/docs/canonicai/chapter9.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+←[32mINFO←[0m:     Started server process [←[36m42808←[0m]
+←[32mINFO←[0m:     Waiting for application startup.
+←[32mINFO←[0m:     Application startup complete.
+←[32mINFO←[0m:     Uvicorn running on ←[1mhttp://0.0.0.0:9000←[0m (Press CTRL+C to quit)
+```
+
+After you see the server started you will have to go to the browser and run this domain:
+* `http://0.0.0.0:9000/docs`
+
+Congratulations, you have successfully use uvicorn to stand up a jaseci_ai_kit server locally.
```
Comment (ypkang) [inline docs/docs/canonicai/chapter9.md:66]: We should include the step to call `actions load remote URL` to then use this server

**Review by ypkang — DISMISSED** [review]

**Review by marsninja — APPROVED** [review]
LGTM @Shawn-Jemmott

**Pull request merged**